seonshine_backend/middleware/validation/restaurantValidate.js

```diff
@@ -0,0 +1,21 @@
+import { httpStatusCodes } from '../../constants/http.js';
+
+export const validateAssignRestaurantDate = (req, res, next) => {
+  const { weekday } = req.body;
+
+  if (!weekday) {
+    return res
+      .status(httpStatusCodes.badRequest)
+      .json({ error: 'Weekday is required' });
+  }
+
+  const isValidWeekday = weekday >= 1 && weekday <= 5;
```

Comment:
This check will not be enough. I will fail with weekday is "3.5". Please check weekday is integer or define constant for weekdays. Ex: const weekdays = {1: "Mondays",...}